3.3/.sti/bin/assemble

```diff
@@ -1,23 +1,66 @@
 #!/bin/bash
 
-# For SCL enablement
-source .bashrc
+# This script uses these environment variables:
+#
+# DISABLE_COLLECTSTATIC: if not empty, inhibits execution of 'manage.py collectstatic'.
 
-set -e
+function assemble() {
+  # For SCL enablement
+  source .bashrc
```

Comment:
>personally I would like to break the assemble function into smaller functions and not call it 'assemble'.

Let's not over-complicate it that much, the functions @rhcarvalho created are readable. If we split it that will be a paint to maintain. It's not that big.